transformers/data/processors/utils.py

```diff
@@ -125,3 +102,215 @@ def _read_tsv(cls, input_file, quotechar=None):
                     line = list(unicode(cell, 'utf-8') for cell in line)
                 lines.append(line)
             return lines
+
+
+class SingleSentenceClassificationProcessor(DataProcessor):
+    """ Generic processor for a single sentence classification data set."""
```

Comment:
I feel that we're using both `sentence` and `sequence` to describe the same thing across the library. We should probably stick to one of them (I have a preference for sequence)

Reply:
yeah i kind of agree

Reply:
I'll unify all of this, s/sentence/sequence

Reply:
Yes, I agree and I like more `sequence` as well